togetherly/lib/providers/person_provider.dart

```diff
@@ -57,16 +66,24 @@ class PersonProvider with ChangeNotifier {
   }
 
   Future<void> refresh() async {
+    _ready = false;
```

Comment:
For this to be reflected in the UI, you'll want to call `notifyListeners` immediately after setting this value (before the async-await stuff below).

Reply:
Done